packages/lib/src/utils/Logger.ts

```diff
@@ -132,14 +134,27 @@ export const Loggy: { [key: string]: any } = {
     }
   },
 
+  _actionToText(action: SpinnerAction): string {
+    switch (action) {
+      case SpinnerAction.Add || SpinnerAction.NonSpinnable:
+        return 'added';
+      case SpinnerAction.Succeed:
+        return 'succeeded';
+      case SpinnerAction.Fail:
+        return 'failed';
+      case SpinnerAction.Update:
+        return 'updated';
+      default:
+        return '';
+    }
+  },
+
   _getColorFor(logType: LogType): string {
     switch (logType) {
-      case LogType.Success:
-        return 'blueBright';
+      case LogType.Err:
+        return 'redBright';
```

Comment:
What happened with LogType.Success? And Fail?

Reply:
I removed them as we were not using them.